src/Package/Impl/project.14.0.json

```diff
@@ -12,6 +12,7 @@
     "Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime": "14.2.25123",
     "Microsoft.VisualStudio.TextManager.Interop.12.0": "12.0.30110",
     "Microsoft.VSSDK.BuildTools": "14.2.25201",
+    "System.Collections.Immutable": "1.1.33-beta",
```

Comment:
The reason for this change is to ensure that we have the right version for CPS. Previously, we didn't depend on it directly, so CPS brought this version via indirect reference. Now that we also bring in ASP.NET Core via NuGet, it has a direct reference on a different version, which was then preferred, and ended up causing breakage when implementing CPS interface, due to mismatches between versions. Explicitly specifying this in projects that reference CPS makes sure that we always use a single, compatible version.